### PR TITLE
Powerbi header

### PR DESCRIPTION
**Pull request by HypnoMama**
- Header does not show logged in sub header for CIT pages
- save to pdf is now link in sub header
- user can select a community/regional district and get a link copied to clipboard for the public view for that page (CIT-232)
-also removed horizontal scroll bar in searchFlyout  (part of cit-348)

- this PR also refactors the login/logout redirect and ensures it is congruent with the page origin.  
- login for coit still shows both idir and bceid options, cit only allows login with idir

- this PR also removes the ability to uncheck email/name from PropDetails2 - email field is disabled and unable to be changed

### CODE REVIEW

File: cit3.0-web/src/components/Page/PowerBi/PowerBi.js [hunk 1]
```diff
@@ -98,17 +108,94 @@ export default function PowerBi(props) {
     }
   }, [reportConfig]);
 
+  useEffect(() => {
+    axios.get("/api/opportunity/options").then((data) => {
+      console.log(data);
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/PowerBi/PowerBi.js:113]: Could we removed this console log?

File: cit3.0-web/src/components/Page/PowerBi/PowerBi.js [hunk 2]
```diff
@@ -174,13 +261,32 @@ export default function PowerBi(props) {
                 }
               },
             ],
+            [
+              "bookmarkApplied",
+              function (event) {
+                console.log("filters", event);
+                window.report.getFilters().then((filters) => {
+                  console.log(filters);
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/PowerBi/PowerBi.js:269]: Could we removed this console log, then, or event handler?

File: cit3.0-web/src/components/Page/PowerBi/PowerBi.js [hunk 3]
```diff
@@ -174,13 +261,32 @@ export default function PowerBi(props) {
...
+                });
+              },
+            ],
+            [
+              "buttonClicked",
+              function (event) {
+                console.log("button", event);
+                window.report.getFilters().then((filters) => {
+                  console.log(filters);
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/PowerBi/PowerBi.js:278]: Could we removed this console log, then, or event handler?

File: cit3.0-web/src/components/Page/PowerBi/PowerBi.js [hunk 4]
```diff
@@ -174,13 +261,32 @@ export default function PowerBi(props) {
...
+              },
+            ],
           ])
         }
         // // Add CSS classes to the div element
         cssClassName="report-style-class"
         // set report object
         getEmbeddedComponent={(embeddedReport) => {
           window.report = embeddedReport;
+          console.log("embed", embeddedReport);
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/PowerBi/PowerBi.js:289]: Could we removed this console log?

File: cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js [hunk 5]
```diff
@@ -27,7 +27,7 @@ import UserFactory from "../../../store/factory/UserFactory";
 export default function PropertyDetails2() {
   const dispatch = useDispatch();
   const [businessContactSync, setBusinessContactSync] = useState(false);
-  const [userInfoSync, setUserInfoSync] = useState(false);
+  // const [userInfoSync, setUserInfoSync] = useState(false);
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js:30]: Could we removed these comments in this file?

File: cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js [hunk 6]
```diff
@@ -253,15 +253,16 @@ export default function PropertyDetails2() {
             </span>
           </div>
         </Row>
-        <Row className="mb-3">
+        {/* <Row className="mb-3">
```
Comment (zacpez) [inline cit3.0-web/src/components/Page/PropertyDetails2/PropertyDetails2.js:256]: Could we removed these comments?

File: cit3.0-web/src/components/Page/PowerBi/PowerBi.js (hunk 3 above)
Comment (HypnoMama) [reply cit3.0-web/src/components/Page/PowerBi/PowerBi.js:278]: Ahhhh  yes sorry, leftover event handlers from testing....forgot to delete when I pivoted lol.  Thank you!

**Review by zacpez — APPROVED** [review]

**Pull request merged**